### ISSUE TITLE
Why so many verbose flags

### ISSUE THREAD

**Issue by sethtroisi** [tensorflow/minigo#372]
Did each contributor add one?

gtp.py:flags.DEFINE_integer('verbose', 1, 'How much debug info to print.')
selfplay.py:flags.DEFINE_integer('verbose', 1, 'How much debug info to print.')
main.py:        verbose: 'How verbose the players should be (see selfplay)' = 1):
rl_loop.py:def selfplay(verbose=2):

**Comment by brilee** [comment]
On this note, i realized that absl.flags defines a --verbosity flags by default, which hooks into python logging module. So everything should probably go through  logging module.

**Comment by amj** [comment]
even better, pretty sure it was just me who added all of them :P

**Comment by brilee** [comment]
this definitely is still a thing, at least in the python codebase. Although I suppose python isn't really used that much anymore for selfplay/eval, which is where the gazillion verbose flags happened.

**Comment by amj** [comment]
@brilee didn't you fix this with the giant flag merge?

**Comment by brilee** [comment]
No, I figured this wasn't strictly necessary for the giant flag rewrite and didn't want to slow it down.

I haven't dug into how absl.logging and absl.flags --verbosity interact. That'd be where to start.
